src/components/Expander/ExpanderGroup.tsx

```diff
@@ -34,7 +40,7 @@ export interface ExpanderGroupProps extends OpenCloseAll {
 
 export interface ExpanderProviderState {
   onClick: (index: number) => void;
```

Comment:
Is this really onClick event? Feels a bit confusing to have click events mixed with internal value change events. It's completely fine this way if used for handling both, user interaction and something that is either programmatic or derived from user action. Could this be renamed to onExpanderOpenChange etc?

Reply:
This is now changed to `onExpanderOpenChange` and seems to be a lot better 👍